File: LLM-Translations/GPT-4/codenet/C++/Python/s783753345.py

while True:
    user_input = input().split()
    a, S, b = int(user_input[0]), user_input[1], int(user_input[2])

    if S == '+':
        print(a + b)
    elif S == '-':
        print(a - b)
    elif S == '*':
        print(a * b)
    elif S == '/':
        print(a / b)
    elif S == '?':
        break

File: LLM-Translations/GPT-4/codenet/C++/Java/s783753345.java

import java.util.Scanner;

public class s783753345 {
    public static void main(String[] args) {
        int a, b;
        char S;
        Scanner input = new Scanner(System.in);

        while (true) {
            a = input.nextInt();
            S = input.next().charAt(0);
            b = input.nextInt();

            if (S == '+') System.out.println(a + b);
            if (S == '-') System.out.println(a - b);
            if (S == '*') System.out.println(a * b);
            if (S == '/') System.out.println(a / b);
            if (S == '?') break;
        }

        input.close();
    }
}

// 
